charts/cbcontainers-agent/cbcontainers-agent-chart/example-values.yaml

```diff
@@ -0,0 +1,188 @@
+# accessToken is the API token used by the agent to communicate with the backend
+accessToken: ""
```

Comment:
Why do we need this if we have accessTokenSecretName?
I think it is less secure as well

Reply:
this allow you to put the access token in the helm chart. I agree that it's somewhat insecure to have it here, so maybe we should leave this here to allow the customers to create everything via the chart, but also we should make it non-required, so that if someone creates and manages the token in a different way to not create the token. WDYT?